src/storage/src/source/kafka.rs

```diff
@@ -184,13 +180,8 @@ impl SourceRender for KafkaSourceConnection {
                         // ensure that librdkafka does not try to perform its own
                         // consumer group balancing, which would wreak havoc with
                         // our careful partition assignment strategy.
-                        "group.id" => format!(
-                            "{}materialize-{}-{}-{}",
-                            group_id_prefix.unwrap_or_else(String::new),
-                            environment_id,
-                            connection_id,
-                            config.id,
-                        ),
+                        "group.id" => group_id.clone(),
+                        "client.id" => format!("{}:{}", group_id, config.worker_id),
```

Comment:
I think we should use `group.id` here as well. Among other things, [throttling](https://kafka.apache.org/documentation/#multitenancy-isolation) is done per client-id... setting your client id at too fine a grain is both not useful and makes it hard to enforce policy on the broker side.

Reply:
didn't know this! can definitely adjust this!